src/main/scala/chisel3/util/SRAM.scala

```diff
@@ -78,6 +81,47 @@ class MemoryReadWritePort[T <: Data](tpe: T, addrWidth: Int, masked: Boolean) ex
   }
 }
 
+/** Description to the SRAM, encoded by the [[chisel3.properties]] API.
+  * User can access it via CIRCT API.
+  */
+@instantiable
+private[chisel3] class SRAMDescription extends Class {
```

Comment:
```suggestion
@instantiable
final class SRAMDescription extends Class {
```
This should be public and should be used in the type of the `description` property in the `SRAMInterface`.

Reply:
Yes, if you are going to use `hierarchy: Property[Path]` in Chisel, this should be public, but if not, private it will be better.